lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp

```diff
@@ -395,8 +395,8 @@ SmallVector<Value> LayoutPropagation::propagateToUsers(Value value,
       continue;
     }
     // Workaround: don't propagate through truncI
-    if (isa<arith::TruncIOp>(user))
-      continue;
+    // if (isa<arith::TruncIOp>(user))
+    //   continue;
```

Comment:
These lines are removed in upstream, maybe cherry-pick changes from there in a separate PR?

Reply:
this one: https://github.com/openai/triton/pull/2585